01-ci/README.md

```diff
@@ -4,7 +4,7 @@ Task #01 - Compile test
 =======================
 ### Description
 
-Execute ./dist/tools/compile_test/compile_test.py
+Open a dummy PR or use a backport PR on the RC and enable the `full_test` label.
```

Comment:
```suggestion
Check the latest release branch bors build ("staging@...").
```

Reply:
done

Reply:
I guess this is obsolete now and should be updated to merge queues...

Reply:
Yeah. Maybe make that CI agnostik: "Check the latest release branch CI build."